### PR TITLE
Updated content to new O365 and SPO admin sites. #3153

### PR DESCRIPTION
**Pull request by j0rt3g4**
Updated content to new O365 and SPO admin sites. #3153

**PSD files were not updated**

Copy review @nenonix @JohanFreelancer9 is required on this one guys.
Thank you in advance!

### CODE REVIEW

File: Teams/guest-access-checklist.md [hunk 1]
```diff
@@ -76,9 +76,9 @@ For more information about controlling who can invite guests, see [Delegate invi
 
 ## Step 3: Configure Office 365 Groups
 
-1. In the Microsoft 365 admin center, go to **Settings** > **Services & Add-ins** > **Office 365 Groups**.
-2. Make sure **Let group members outside the organization access group content** is set to **On**. If this setting is turned off, guests won't be able to access any group content.
-3. Make sure **Let group owners add people outside the organization to groups** is set to **On**. If this setting is turned off, Team owners won't be able to add new guests. At a minimum, this setting must be On to support guest access.
+1. In the new Microsoft 365 admin center, go to **Show All** > **Settings** > **Services & Add-ins** > **Office 365 Groups**.
+2. Make sure the checkbox **Let group members outside the organization access group content** is **Checked**. If this setting is **Unchecked**, guests won't be able to access any group content.
```
Comment (JohanFreelancer9) [inline Teams/guest-access-checklist.md]:
```suggestion
2. Make sure that the checkbox **Let group members outside the organization access group content** is **Checked**. If this setting is **Unchecked**, guests won't be able to access any group content.
```

File: Teams/guest-access-checklist.md [hunk 2]
```diff
@@ -76,9 +76,9 @@ For more information about controlling who can invite guests, see [Delegate invi
...
+3. Make sure the checkbox **Let group owners add people outside the organization to groups** is **Checked**. If this setting is **Unchecked**, Team owners won't be able to add new guests. At a minimum, this setting must be On to support guest access.
```
Comment (JohanFreelancer9) [inline Teams/guest-access-checklist.md]:
```suggestion
3. Make sure that the checkbox **Let group owners add people outside the organization to groups** is **Checked**. If this setting is **Unchecked**, Team owners won't be able to add new guests. At a minimum, this setting must be **On** to support guest access.
```

File: Teams/guest-access-checklist.md [hunk 3]
```diff
@@ -110,11 +110,18 @@ Make sure that users can add guests. Here's how:
 This one's a bit of a brain teaser. Guest access in Teams doesn't work if the **Don't allow sharing outside your organization** setting is selected in the SharePoint admin center.
 
 1. Sign in to the Microsoft 365 admin center.
-2. Click **Admin center**, and then select **SharePoint**.
-3. In the SharePoint admin center, select **Sharing**.
-4. Make sure the option for **Don’t allow sharing outside your organization** is *not* selected.
+2. Click **Show All** and under **Admin Centers**, select  **SharePoint**.
+3. On the new Sharepoint Admin Center,  under **Sites** > select **Active Sites**.
```
Comment (JohanFreelancer9) [inline Teams/guest-access-checklist.md]:
```suggestion
3. In the new SharePoint Admin Center,  under **Sites**, select **Active Sites**.
```

**Comment by j0rt3g4** [comment]
Thank you @JohanFreelancer9

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**